tests/e2e/specs/tree.js

```diff
@@ -108,8 +108,12 @@ describe('Tree component', () => {
           .get('.node-data-cyclepoint')
           .should('be.visible')
         cy.window().its('app.$workflowService.subscriptions').then(subscriptions => {
-          // FIXME: likely wrong, but to be fixed later in a follow-up PR to housekeep subscriptions
```

Comment:
There were several tests like this one. Where we were checking for the wrong number of subscriptions. That's because the `MockWorkflowService` still didn't handle starting/stopping subscriptions. Since we are not using it any longer, now we are able to confirm the subscriptions are created and destroyed as expected :tada: (it was unintentionally fixed here :sweat_smile: )